src/pages/community-champions.js

```diff
@@ -323,6 +323,7 @@ const HowToQualifyCardContainer = styled('div')`
 const HowToQualifyCardTitle = styled(H5)`
     margin-bottom: ${size.xsmall};
     margin-top: ${size.default};
+    word-wrap: break-word;
```

Comment:
need this bc the title in the `Participate` card overflows on a Galaxy Fold (and other phones with narrow screens) otherwise

Reply:
Nice! good find!